internal/core/command/put_test.go

```diff
@@ -73,3 +82,45 @@ func TestNewPutCommandBody(t *testing.T) {
 		t.Error("Failed to verify the request body contents")
 	}
 }
+
+func TestNewPutCommandContentType(t *testing.T) {
```

Comment:
Please refer to my other comment regarding adding test coverage for different negative test cases.